src/WebJobs.Script/Config/CompatibilityModeOptions.cs

```diff
@@ -1,10 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-namespace Microsoft.Azure.WebJobs.Script.Configuration
-{
-    internal class CompatibilityModeOptions
-    {
-        public bool IsV2CompatibilityModeEnabled { get; set; }
```

Comment:
We still have a global flag with a method we can use to check this, correct?

Reply:
Yes, `_environment.IsV2CompatibilityMode()`.